sdk/core/azure-core/inc/azure/core/http/http.hpp

```diff
@@ -175,6 +175,30 @@ namespace Azure { namespace Core { namespace Http {
     NetworkAuthenticationRequired = 511, ///< HTTP 511 Network Authentication Required.
   };
 
+  /**
+   * @brief Defines a range of bytes within an HTTP resource, starting at an `Offset` and ending at
+   * `Offset + Length - 1` inclusively.
+   *
+   */
+  struct Range
+  {
+    /**
+     * @brief The starting point of the HTTP Range.
+     *
+     */
+    int64_t Offset = 0;
```

Comment:
I think we're missing `#include <stdint.h>` for `int64_t`.

Reply:
I prefer `#include <cstdint>`

Reply:
Should we put this in the guidelines? Is `cstdint` the preferred/idiomatic way of doing it in C++ and is there something wrong about using `stdint.h`? I saw some posts in SO favoring cstdint and leaving stdint.h for C programs. @CaseyCarter - worth being consistent on or does it largely not matter?